parse_config/Block.cpp

```diff
@@ -0,0 +1,27 @@
+#include "Block.hpp"
+#include <iostream>
+
+Block::Block(string _type) : type(_type) {}
+
+/*지시어에 argument들을 vector<string> 으로 저장해 return*/
+vector<string>	Block::getValue(vector<string>::iterator& word)
+{
```

Comment:
```suggestion
vector<string>	Block::getValue(vector<string>::iterator& word) {
```
이전에 얘기 했던 컨벤션으로 적용하면 좋을거 같아요! 다른부분들도 참고해서 수정해주시면 감사하겠습니다!

Reply:
알겠습니다!